draft-ietf-rats-architecture.md

```diff
@@ -759,6 +759,16 @@ could establish trust in a Verifier as discussed above.
 
 ## Attester
 
+The manufacturer of the Attester arranges for the Attester to be provisioned with key material.
+As an Endorser, this manufacturer also arranges for the Verifier to be able to verify signatures from this keying material.
+
+The key material is typically in the form of asymmetric keypairs (e.g., an RSA or ECDSA private key
+and a manufacturer signed IDevID certificate) secured in the Attester, with an appropriate trust anchor provided to the Verifier.
+But, these Endorsements may come in many other forms including databases of public keys (rather than certificates), or even carefully secured lists of symmetric keys.
```

Comment:
"these Endorsements" doesn't make sense in context since you haven't mentioned any Endorsements, and the trust anchor provisioned in the Verifier need not be from an Endorsement